lib/src/screens/video_details_screen/video_details_screen.dart

```diff
@@ -195,6 +263,16 @@ class _VideoDetailsScreenState extends State<VideoDetailsScreen> {
                 child: AppBar(
                   title: Text(details.title),
                   actions: [
+                    IconButton(
+                      onPressed: () {
+                        Share.share(
+                            'Watch it on 3Speak https://3speak.tv/watch?v=' +
+                                details.owner +
+                                "/" +
+                                details.permlink);
+                      },
+                      icon: const Icon(Icons.share),
+                    ),
```

Comment:
- Only these lines are eligible for this pull request. 
- Please remove other changes from pull request.
- Make sure to attach a screenshot for visual comparison 🙏
- Upvotes & Downvotes related UI is not suited for this pull reqeuest.
- Use `InkWell` instead of `GestureDetector`